src/mollie.js

```diff
@@ -0,0 +1,37 @@
+import fs from 'fs';
+import path from 'path';
+import https from 'https';
+import assign from 'lodash/assign';
+
+import createHttpClient from './utils/create-http-client';
+import createMollieApi from './create-mollie-api';
+import cert from './cacert.pem';
```

Comment:
it's the CA cert, you normally have these on your computer and these store the things needed to check if a cert is valid, so by including it here it makes it so they're not dependent on mayor CAs, so the TLS connection will always work regardless of the CA certs installed on the server.
My explaination is not perfect but here's the wiki article on CAs
https://en.wikipedia.org/wiki/Certificate_authority